aiohttp/web_urldispatcher.py

```diff
@@ -28,12 +29,25 @@
 __all__ = ('UrlDispatcher', 'UrlMappingMatchInfo',
            'AbstractResource', 'Resource', 'PlainResource', 'DynamicResource',
            'AbstractRoute', 'ResourceRoute',
-           'StaticResource', 'View')
+           'StaticResource', 'View', 'RoutesDef',
+           'head', 'get', 'post', 'patch', 'put', 'delete', 'route')
 
 HTTP_METHOD_RE = re.compile(r"^[0-9A-Za-z!#\$%&'\*\+\-\.\^_`\|~]+$")
 PATH_SEP = re.escape('/')
 
 
+class RouteDef(namedtuple('_RouteDef', 'method, path, handler, kwargs')):
```

Comment:
Because we already use `Route` name for routes added by `app.router.add_route()` call.
`RouteDef` is a scratch for non-added-yet route definition